app/views/admin/people/show.html.slim

```diff
@@ -0,0 +1,40 @@
+- content_for :title
+  = @person.name
+- content_for :styles
+  = stylesheet_link_tag "admin/media/media", media: "all"
```

Comment:
This should not be necessary as we are now including these globally. Please try without and if it works, please also remove from organizations#show template.